dataSources.py

```diff
@@ -1,98 +1,264 @@
+# groupNames
+community = "Community support"
+covid = "COVID vulnerability"
+demographics = "Demographics"
+bias = "Risk of bias in our sources"
```

Comment:
Since all the objects defined in this module are very likely to be `Constants`, they should be all written in UPPPERCASE ;)

Commenting here but applies to the whole file.

Reply:
@CMorenoStokoe please do this with automatic refactoring.. not manually - use PyCharm (if you have it) or VSCode (although not as good as PyCharm in Refactoring) to make sure this change won't break other files where you're actually importing these constants ;)